vSphere certification and other provider changes (#470)
kubev2v/forklift-documentation: documentation/modules/adding-source-provider.adoc

* *URL*: URL of the SDK endpoint of the vCenter on which the source VM is mounted. Ensure that the URL includes the `sdk` path, usually `/sdk`. For example, `https://vCenter-host-example.com/sdk`. If a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate.
* *VDDK init image*: `VDDKInitImage` path. It is strongly recommended to create a VDDK init image to accelerate migrations. For more information, see xref:../master.adoc#creating-vddk-image_mtv[Creating a VDDK image].
* *Username*: vCenter user. For example, `user@vsphere.local`.
* *Password*: vCenter user password.
* *SHA-1 fingerprint*: The provider currently requires the SHA-1 fingerprint of the vCenter Server's TLS certificate in all circumstances. vSphere calls this the server's thumbprint.
// * *SHA-1 fingerprint*: The provider currently requires the SHA-1 fingerprint of the vCenter Server's TLS certificate in all circumstances. vSphere calls this the server's thumbprint.
+

. Choose one of the following options for validating CA certificates:

** *Skip certificate validation* : Migrate without validating a CA certificate.

> ** *Use the system CA certificates*: Migrate after validating the system CA certificates.

Reviewer: no need to delete this line:
if you have:
  - skip certicate validation - off
  - custom ca certificate - empty

it will use the system ca certificate

.. To skip certificate validation, select the *Skip certificate validation* check box.

> .. To validate a custom CA certificate, leave the *Skip certificate validation* check box cleared and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.

Reviewer: ```suggestion
.. To use custom CA certificate, leave the *Skip certificate validation* check box cleared and enter a custom CA certificate in the input box.
```

note I:
"To validate a custom CA certificate," is misleading, it's to "not skip certificate validation" when performing migration from a cluster

note II:
you can "Enter a custom CA certificate" by typing it manually in the input box, choosing a file with the certificate, or fetching it from URL (fetching from a URL option is currently only of vSphere)

Reviewer: note III:
when choosing to get the CA certificate by fetching it from URL, you will also need to "verify" the new CA certificate by checking it's fingerprint, and only after verifying manually it's a secure certificate, you can use it for secure migration
